examples/AzureSDKDemoSwift/Source/Common.swift

```diff
@@ -210,6 +210,8 @@ extension TransferState {
             return .systemRed
         case .deleted:
             return .systemRed
+        @unknown default:
```

Comment:
Turns out iOS has this solution, making it unnecessary to have either a `.unknown` or use the implicitly unwrapped optional.

Reply:
Interesting, will read up on this!

Reply:
https://www.avanderlee.com/swift/unknown-default-enums-in-swift/

Reply:
I understand how this works from the consuming side, but I think I'll have to play around with it to understand how it impacts our code which creates an enum from its stored rawValue.